docs/guide/SecurityBC.md

```diff
@@ -273,3 +273,234 @@ Here is the response that comes back:
 "allowed": true/false
 }
 ```
+
+## Ory Oathkeeper – implementation detail
+*(Configuring Oathkeeper for BizOps)*
+[ORY Oathkeeper](https://www.ory.sh/oathkeeper/docs/next/) authorizes incoming HTTP requests. It can be the Policy Enforcement Point in your cloud architecture, i.e. a reverse proxy in front of your upstream API or web server that rejects unauthorized requests and forwards authorized ones to your server. If you want to use another API Gateway (Kong, Nginx, Envoy, AWS API Gateway, ...), Oathkeeper can also plug into that and act as its Policy Decision Point.
+
+The Oathkeeper helm chart is described at [ORY Oathkeeper Helm Chart | k8s](https://k8s.ory.sh/helm/oathkeeper.html) and defined in [k8s/helm/charts/oathkeeper at master · ory/k8s · GitHub](https://github.com/ory/k8s/tree/master/helm/charts/oathkeeper) The Ory helm repository is documented at [ORY Helm Charts | k8s](https://k8s.ory.sh/helm/). The Oathkeeper configuration reference is at [Configuration | ORY Oathkeeper](https://www.ory.sh/oathkeeper/docs/reference/configuration) but note that the Helm chart values has different ways of doing certain things. Also, every configuration value can be overridden with environment variables. 
+
+The Oathkeeper helm chart deploys two key components into Kubernetes: Oathkeeper itself, and the Oathkeeper Maester. Oathkeeper is stateless and config-driven, and seamlessly reloads itself with zero downtime anytime that config changes. Oathkeeper Maester is a controller for the Rule CRD, and composes the Rule objects in Kubernetes into a single complete rules file that is loaded by Oathkeeper.  By default, that’s a ConfigMap that Oathkeeper mounts, but it can also be set to run as a sidecar and use a shared mount.

```

Comment:
```suggestion
The Oathkeeper Helm chart deploys two key components into Kubernetes: Oathkeeper itself, and the Oathkeeper Maester. Oathkeeper is stateless and config-driven, and seamlessly reloads itself with zero downtime, anytime that config changes. Oathkeeper Maester is a controller for the Rule CRD, and composes the Rule objects in Kubernetes into a single complete rules file that is loaded by Oathkeeper.  By default, that is a ConfigMap that Oathkeeper mounts, but it can also be set to run as a sidecar and use a shared mount.

```